gcli/src/meta/executor.rs

```diff
@@ -140,7 +140,7 @@ mod env {
                 "gr_gas_available" => func!(store, i32),
                 "gr_leave" => func!(store),
                 "gr_message_id" => func!(store, i32),
-                "gr_out_of_gas" => func!(store),
+                "gr_system_break" => func!(store, u64),
```

Comment:
> @clearloop can you look if this will cause any problems when the user wants to get metadata from a program that has not yet updated the instrumented version?

It will cause such problem, but for me its not necessary to support. Re-instrumentation will always be on first access to program on-chain

Reply:
I see that gcli uses the local wasm file and then applies instrumentation to it. In this case everything should work.